packages/cli/index.js

```diff
@@ -212,10 +241,68 @@ async function runAppPackageJsonScript (appName, script) {
     return ''
 }
 
+/**
+ * Takes all environment default variables from {fromPath} env file (if exists), and moves them to {toPath} env file
+ * If opts.override = true, existing keys in toPath will be overwritten, otherwise they will stay the same
+ * @param {string} fromPath
+ * @param {string} toPath
+ * @param {{override: boolean}} opts
+ * @return {Promise<void>}
+ */
+async function moveEnv (fromPath, toPath, opts = { override: true }) {
```

Comment:
mb `copyEnv`/`fill...`/`update...`?

In my understanding it works like this :)

action   |    before    |    after
-|-|-
move     |  [1] -> [   ]   |    [   ] -> [1]
copy      |  [1] -> [   ]   |    [1] -> [1]

Reply:
Renamed to `copyEnv`